utils/spec-bundle.js

```diff
@@ -30,6 +30,8 @@ require('zone.js/dist/sync-test');
 require('zone.js/dist/proxy');
 require('zone.js/dist/jasmine-patch');
 
+require('reflect-metadata');
```

Comment:
@Blackbaud-MikitaYankouski Mind providing more information on how this affects your unit tests?

Reply:
Without this line the unit tests do not run and the code coverage results aren't printed so our build ends up failing as well.